app/lib/actions/pulp3/orchestration/orphan_cleanup/remove_orphans.rb

```diff
@@ -0,0 +1,14 @@
+module Actions
+  module Pulp3
+    module Orchestration
+      module OrphanCleanup
+        class RemoveOrphans < Pulp::Abstract
+          def plan(proxy)
+            plan_action(Actions::Pulp3::Orchestration::Repository::RemoveOrphans, proxy) if proxy == SmartProxy.pulp_master
```

Comment:
this Repository::RemoveOrphans orchestration action isn't really needed, lets pull the version removal out of it and put it here.

Reply:
look at the 'tree' i wrote out, i think thats a good 'guide' for what we want the action structure to look like IMO